conditions.md

```diff
@@ -9,11 +9,14 @@ Description of conditions https://github.com/kubernetes/community/blob/master/co
 ## Conditions in openstack-k8s-operators podified controlplane operators
 * At the beginning of the reconciliation loop, controller adds a `Ready` condition which reflects the overall state of the service with `Status=Unknown`
 * When a controller performs a task, before it starts a step condition gets initialized to `Status=Unknown` e.g. DBReady, DBSync, Endpoint, ServiceRegistered, …
+* Conditions should always be re-initialized at the beginning of each reconcile. This is to ensure that changes to the spec
+  will reset conditions accordingly.
```

Comment:
please note about not changing transition times if the condition value doesn't change